controllers/util.go

```diff
@@ -0,0 +1,79 @@
+package controllers
+
+import (
+	"slices"
+
+	synv1alpha1 "github.com/projectsyn/lieutenant-operator/api/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+)
+
+func envVarIndex(name string, list *[]synv1alpha1.EnvVar) int {
```

Comment:
There's `slices.IndexFunc` for less boilerplate https://pkg.go.dev/golang.org/x/exp/slices#IndexFunc